Makefile

```diff
@@ -134,9 +141,15 @@ souffle:
                 sudo apt update; \
                 sudo apt install souffle; \
                 ;; \
+			"Darwin") \
```

Comment:
I checked on my machine and it seems that this block for `"Darwin"` is indented using tabs instead of spaces.

Reply:
See commit [1cdef4e](https://github.com/oracle/macaron/pull/383/commits/1cdef4e8fde94b7fa027149483caf90ea9441a80)